tests/test_compositional_attention.py

```diff
@@ -29,10 +29,10 @@
 assert ATTENTION_REGISTRY.keys(), "Attention layers should have been registered"
 
 
-@pytest.mark.parametrize("attn_dropout", [0.0, 0.3])
+@pytest.mark.parametrize("attn_dropout", [0.3])
```

Comment:
same as before, we should keep 0 because different codepath down the line ([see](https://github.com/facebookresearch/xformers/blob/main/xformers/triton/dropout.py#L171) for instance)